### ISSUE TITLE
Think of better solution to dropdown menu on Contact us form

### ISSUE THREAD

**Issue by anikbrazeau** [cds-snc/notification-api#976]
Dropdown menu should be last resort.

<img width="448" alt="Screen Shot 2020-06-19 at 10 09 21 AM" src="https://user-images.githubusercontent.com/38330843/85141580-fa7ff000-b214-11ea-946d-33da0baf9d42.png">

<img width="451" alt="Screen Shot 2020-06-19 at 10 09 07 AM" src="https://user-images.githubusercontent.com/38330843/85141584-fb188680-b214-11ea-908d-16a47f860d11.png">

There are only 4 options within it. Might there be a better way to present information?

Purpose: 
-guides those who write as to what sort of questions they can ask us and what sort of support we have for them
- helps to sort messages to go to the right person to address them in Freshdesk

**Comment by anikbrazeau** [comment]
From Bryan:

 'other' is just inflated with everything and we really should have made a go live request option in the drop down.

Removing the drop down may break the API to Fresh-desk though because it expects the information so it has to be done with care.

If it is technically accessible lets leave it for now. If there is a quick fix you want to do like cut the dropdown and put in hints, let me know we can do it for Beta. 

Otherwise I don't mind leaving this until we revamp support process and stuff. (I am trying to get us a CRM product)

**Comment by anikbrazeau** [comment]
Consider leaving it for now since it is technically accessible (the way it is built), but if we don't need it anymore as functionality to filter where the messages go/who replies to them etc, but only to guide users as to what they can write us about, then we could probably just use a hint.

When we revisit, these prompts could be more specific to common issues such as :
-- request to go live
-- get technical support
